lib/sles4sap_publiccloud.pm

```diff
@@ -5,30 +5,43 @@
 #
 # Summary: Library used for SLES4SAP publicccloud deployment and tests
 
-use base 'publiccloud::basetest';
 package sles4sap_publiccloud;
+
+use base 'publiccloud::basetest';
 use strict;
 use warnings FATAL => 'all';
+use Exporter 'import';
+use version_utils 'is_sle';
```

Comment:
are `is_sle` and line 20 `Data::Dumper` used?

Reply:
Thanks, good catch. 
Removed.